tools/ci.py

```diff
@@ -967,8 +967,8 @@ def get_ci_deps_matrix(ctx: Context):
 
     _matrix = {
         "linux": [
-            {"distro-slug": "centos-7", "arch": "x86_64"},
-            {"distro-slug": "centos-7-arm64", "arch": "arm64"},
+            {"distro-slug": "rockylinux-8", "arch": "x86_64"},
+            {"distro-slug": "rockylinux-8-arm64", "arch": "arm64"},
```

Comment:
@dwoz @dmurphy18 

I'll need confirmation that, with the deprecation of CentOS 7, that we want to bump to Rocky Linux 8 here. Since this originally had CentOS 7 hardcoded for this CI job, I went with the assumption that we should go with the oldest supported version of Rocky Linux (8) as the replacement for this step.

If we'd rather go with Rocky Linux 9, instead, then we can apply the following suggestion:

```suggestion
            {"distro-slug": "rockylinux-9", "arch": "x86_64"},
            {"distro-slug": "rockylinux-9-arm64", "arch": "arm64"},
```

Reply:
Register article last week had that maintenance on 8 was about to stop but they would support it till end of the decade or thereabouts, so maybe 9 is the better way to go, see https://www.theregister.com/2024/05/29/rhel_and_alma_8_10/